lib/3scale_toolbox/crds/backend_parser.rb

```diff
@@ -0,0 +1,55 @@
+module ThreeScaleToolbox
+  module CRD
+    class BackendParser
+      Metric = Struct.new(:system_name, :friendly_name, :description, :unit)
```

Comment:
Here we are defining our own custom structs.

There are their corresponding 'Entity' classes, can't we reuse them? some classes on the model will be using this custom structs and others will be using the entities one, which is kind of confusing.

Reply:
This `Metric` struct is private for the `BackendParser` and should not be used externally. 

The entities are interface to external remote objects and have methods and required input parameters (like remote) that do not make sense for this structs. They only share the list of attributes.

Reply:
:+1: